Assets/Scripts/GameCharacter/ICharacter.cs

```diff
@@ -1,5 +0,0 @@
-
-public interface ICharacter 
```

Comment:
Why this interface is empty? Is it just a marker for something?
Could you use it adding some of the properties you add to Hero implementation?